app/controllers/surveys_controller.rb

```diff
@@ -1,4 +1,35 @@
 class SurveysController < ApplicationController
+
   def send_survey
+    respondent_ids = JSON.parse(params[:respondent_ids]).uniq
+
+    begin
+      survey_id = params[:id]
+      survey = Survey.find(survey_id)
+    rescue ActiveRecord::RecordNotFound
+      return render :json => {:error => :survey_not_found}, :status => :bad_request
+    end
+
+    if survey.first_question.nil?
+      return render :json => {:error => :survey_missing_first_question}, :status => :bad_request
+    end
+
+    if respondent_ids.empty?
```

Comment:
I think we should return some sort of message to the front end, because i assume they would also have some logic preventing sending a survey to nobody, so if this does happen something went wrong.